diff -ur Brayns-ffc3949/apps/BraynsViewer/CMakeLists.txt Brayns-ffc3949.new2/apps/BraynsViewer/CMakeLists.txt
--- Brayns-ffc3949/apps/BraynsViewer/CMakeLists.txt	1970-01-01 01:00:01.000000000 +0100
+++ Brayns-ffc3949.new2/apps/BraynsViewer/CMakeLists.txt	2018-02-06 18:04:27.483969725 +0100
@@ -11,7 +11,7 @@
 )
 
 set(BRAYNSVIEWER_LINK_LIBRARIES
-  PUBLIC brayns braynsCommon braynsIO braynsParameters braynsUI
+  PUBLIC brayns braynsCommon braynsIO braynsParameters braynsUI ${OPENGL_gl_LIBRARY}
 )
 
 common_application(braynsViewer)
