From 499d9bf57d88db84107310f01c84735ca3636a1d Mon Sep 17 00:00:00 2001
From: devresse <adrien.devresse@epfl.ch>
Date: Fri, 10 Mar 2017 15:21:31 +0000
Subject: [PATCH] Fix API break issue with last version of vmmlib

---
 BlueRepairSDK/src/repair.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/BlueRepairSDK/src/repair.cpp b/BlueRepairSDK/src/repair.cpp
index 077b45a..4b28ba1 100644
--- a/BlueRepairSDK/src/repair.cpp
+++ b/BlueRepairSDK/src/repair.cpp
@@ -682,7 +682,7 @@ int main (int argc, char *argv[])
   grown = cut;
 
   // REPAIR
-  TranslatedRotation tr(Vector3f::ZERO, bbp::Rotation_3D(0.0, 0.0, 0.0), Vector3f::ZERO);
+  TranslatedRotation tr(Vector3f::zero(), bbp::Rotation_3D(0.0, 0.0, 0.0), Vector3f::zero());
   Micron y_extent = 1.1 * max_y_extent(newMorph.soma(), tr);
   Micron cylindric_extent =
       max_cylindric_extent(*newMorph.dendrites().begin());
@@ -792,7 +792,7 @@ int main (int argc, char *argv[])
               bbp::Orientation(axis, 180 * alpha / bbp::PI));
           post = rotation * post;
 
-          Vector3f offsetFromOrigin = Vector3f::ZERO - segmentStartPoint(from);
+          Vector3f offsetFromOrigin = Vector3f::zero() - segmentStartPoint(from);
 
           const TranslatedRotation translation(offsetFromOrigin,
                                                rotation,
-- 
1.9.1

