diff -ur BGLibPy-4658871/setup.py BGLibPy-4658871.new/setup.py
--- BGLibPy-4658871/setup.py	1980-01-02 00:00:00.000000000 +0100
+++ BGLibPy-4658871.new/setup.py	2018-01-16 17:02:06.160911501 +0100
@@ -14,9 +14,9 @@
     description="The Pythonic Blue Brain simulator access",
     license="BBP-internal-confidential",
     dependency_links=["https://bbpteam.epfl.ch/repository/devpi/bbprelman/dev/+simple/bluepy/"],
-    extras_require={"bbp": ["bluepy[bbp]>=0.11.10.dev1", "brain", "pybinreports>=0.4"]},
+    extras_require={"bbp": ["bluepy[bbp]>=0.11.10.dev1", "pybinreports>=0.4"]},
     install_requires=["bluepy>=0.11.10.dev1", "bluepy-configfile>=0.1.2.dev1"],
-    tests_require=["bluepy[bbp]>=0.11.10.dev1", "brain"],
+    tests_require=["bluepy[bbp]>=0.11.10.dev1"],
     keywords=(
         'computational neuroscience',
         'simulation',
