diff --git a/CMakeLists.txt b/CMakeLists.txt
index b2465465..202db89a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -186,7 +186,7 @@ if ( HAVE_PYTHON )
     COMMAND ${CMAKE_COMMAND} -E env
       NEST_PATH="${CMAKE_INSTALL_FULL_BINDIR}"
       PYTHON="${PYTHON}"
-      NEST_PYTHONPATH="${CMAKE_INSTALL_PREFIX}/${PYEXECDIR}"
+      NEST_PYTHONPATH="${PYEXECDIR}"
       ${CMAKE_INSTALL_FULL_DATADIR}/extras/do_tests.sh --test-pynest --source-dir="${PROJECT_SOURCE_DIR}"
     WORKING_DIRECTORY "${PROJECT_BINARY_DIR}"
     COMMENT "Execute NEST's testsuite...."
diff --git a/cmake/ConfigureSummary.cmake b/cmake/ConfigureSummary.cmake
index baebe0d0..c29183aa 100644
--- a/cmake/ConfigureSummary.cmake
+++ b/cmake/ConfigureSummary.cmake
@@ -157,7 +157,7 @@ function( NEST_PRINT_CONFIG_SUMMARY )
   message( "" )
   if ( HAVE_PYTHON )
     message( "PyNEST will be installed to:" )
-    message( "    ${CMAKE_INSTALL_PREFIX}/${PYEXECDIR}" )
+    message( "    ${PYEXECDIR}" )
     message( "" )
   endif ()
   message( "To set necessary environment variables, add the following line" )
diff --git a/extras/ConnPlotter/CMakeLists.txt b/extras/ConnPlotter/CMakeLists.txt
index 1f7c2b32..aad4680c 100644
--- a/extras/ConnPlotter/CMakeLists.txt
+++ b/extras/ConnPlotter/CMakeLists.txt
@@ -21,7 +21,7 @@ if ( HAVE_PYTHON )
   install( CODE "execute_process(
   COMMAND ${PYTHON} setup.py build --build-base=${CMAKE_CURRENT_BINARY_DIR}/build
                              install --prefix=${CMAKE_INSTALL_PREFIX}
-                                     --install-lib=${CMAKE_INSTALL_PREFIX}/${PYEXECDIR}
+                                     --install-lib=${PYEXECDIR}
                                      --install-scripts=${CMAKE_INSTALL_FULL_BINDIR}
                                      --install-data=${CMAKE_INSTALL_FULL_DATADIR}
     WORKING_DIRECTORY \"${CMAKE_CURRENT_SOURCE_DIR}\")"
diff --git a/extras/nest_vars.sh.in b/extras/nest_vars.sh.in
index 61924310..d636d96a 100644
--- a/extras/nest_vars.sh.in
+++ b/extras/nest_vars.sh.in
@@ -13,7 +13,7 @@ export NEST_DOC_DIR=$NEST_INSTALL_DIR/@CMAKE_INSTALL_DOCDIR@
 export NEST_MODULE_PATH=$NEST_INSTALL_DIR/@CMAKE_INSTALL_LIBDIR@/nest
 
 # The path where the PyNEST bindings are installed.
-export NEST_PYTHON_PREFIX=$NEST_INSTALL_DIR/@PYEXECDIR@
+export NEST_PYTHON_PREFIX=@PYEXECDIR@
 export PYTHONPATH=$NEST_PYTHON_PREFIX:$PYTHONPATH
 
 # Make nest / sli /... executables visible.
diff --git a/pynest/CMakeLists.txt b/pynest/CMakeLists.txt
index 76482e01..4e228d31 100644
--- a/pynest/CMakeLists.txt
+++ b/pynest/CMakeLists.txt
@@ -75,7 +75,7 @@ if ( HAVE_PYTHON )
   install( CODE "execute_process(
     COMMAND ${PYTHON} setup.py build --build-base=${CMAKE_CURRENT_BINARY_DIR}/build
                                install --prefix=${CMAKE_INSTALL_PREFIX}
-                                       --install-lib=${CMAKE_INSTALL_PREFIX}/${PYEXECDIR}
+                                       --install-lib=${PYEXECDIR}
                                        --install-scripts=${CMAKE_INSTALL_FULL_BINDIR}
                                        --install-data=${CMAKE_INSTALL_FULL_DATADIR}
     WORKING_DIRECTORY \"${CMAKE_CURRENT_BINARY_DIR}\")"
@@ -94,7 +94,7 @@ if ( HAVE_PYTHON )
     add_test( NAME PyNEST
         COMMAND ${NOSETESTS} -v --with-xunit
         --xunit-file=${PROJECT_BINARY_DIR}/reports/pynest_tests.xml
-        ${CMAKE_INSTALL_PREFIX}/${PYEXECDIR}/nest/tests
+        ${PYEXECDIR}/nest/tests
         )
   endif ()
 endif ()
diff --git a/testsuite/do_tests.sh.in b/testsuite/do_tests.sh.in
index 46afde51..a1ceca58 100755
--- a/testsuite/do_tests.sh.in
+++ b/testsuite/do_tests.sh.in
@@ -728,7 +728,7 @@ if test "x${TEST_PYNEST}" = xtrue ; then
         echo "  Using nosetests."
         echo
 
-        nosetests -v --with-xunit --xunit-file=${TEST_OUTDIR}/pynest_tests.xml ${NEST_PYTHON_PREFIX:-@CMAKE_INSTALL_PREFIX@/@PYEXECDIR@}/nest/tests ${NEST_PYTHON_PREFIX:-@CMAKE_INSTALL_PREFIX@/@PYEXECDIR@}/nest/topology/tests 2>&1 \
+        nosetests -v --with-xunit --xunit-file=${TEST_OUTDIR}/pynest_tests.xml ${NEST_PYTHON_PREFIX:-@PYEXECDIR@}/nest/tests ${NEST_PYTHON_PREFIX:-@PYEXECDIR@}/nest/topology/tests 2>&1 \
             | tee -a "${TEST_LOGFILE}" | grep -i --line-buffered "\.\.\. ok\|fail\|skip\|error" | sed 's/^/  /'
 
         PYNEST_TEST_TOTAL="$(  tail -n 3 ${TEST_LOGFILE} | grep Ran | cut -d' ' -f2 )"
diff --git a/topology/CMakeLists.txt b/topology/CMakeLists.txt
index da4fb703..6656f483 100644
--- a/topology/CMakeLists.txt
+++ b/topology/CMakeLists.txt
@@ -84,7 +84,7 @@ if ( HAVE_PYTHON )
   install( CODE "execute_process(
     COMMAND ${PYTHON} setup.py build --build-base=${CMAKE_CURRENT_BINARY_DIR}/build
                                install --prefix=${CMAKE_INSTALL_PREFIX}
-                                       --install-lib=${CMAKE_INSTALL_PREFIX}/${PYEXECDIR}
+                                       --install-lib=${PYEXECDIR}
                                        --install-scripts=${CMAKE_INSTALL_FULL_BINDIR}
                                        --install-data=${CMAKE_INSTALL_FULL_DATADIR}
     WORKING_DIRECTORY \"${CMAKE_CURRENT_BINARY_DIR}\")"
