diff -ur blis-898614a555ea0aa7de4ca07bb3cb8f5708b6a002-src/config/reference/bli_config.h blis-898614a555ea0aa7de4ca07bb3cb8f5708b6a002-src.new/config/reference/bli_config.h
--- blis-898614a555ea0aa7de4ca07bb3cb8f5708b6a002-src/config/reference/bli_config.h	2016-08-09 12:00:17.221379378 +0200
+++ blis-898614a555ea0aa7de4ca07bb3cb8f5708b6a002-src.new/config/reference/bli_config.h	2016-08-09 11:56:05.331237186 +0200
@@ -36,6 +36,7 @@
 #define BLIS_CONFIG_H
 
 
+#define BLIS_ENABLE_CBLAS
 
 #endif
 
