From d69019eb86022b65b5443b3720c0a6bbe9226ea3 Mon Sep 17 00:00:00 2001
From: Tristan Carel <tristan.carel@epfl.ch>
Date: Wed, 29 Nov 2017 16:10:58 +0100
Subject: [PATCH] Do not depends on enum34 with Python 3.5 or higher

---
 pyspark/setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyspark/setup.py b/pyspark/setup.py
index 6bbbd62..dd64ed4 100644
--- a/pyspark/setup.py
+++ b/pyspark/setup.py
@@ -94,7 +94,6 @@ def setup_package():
         install_requires=[
             'future',
             'docopt',
-            'enum34;python_version<"3.4"',
             'numpy',
             'lazy-property',
             'py4j',
-- 
2.15.0

