diff -ur '--exclude=CMake' RTNeuron-78dca3d/rtneuron/rtneuron/SceneImpl.cpp RTNeuron-78dca3d.new/rtneuron/rtneuron/SceneImpl.cpp
--- RTNeuron-78dca3d/rtneuron/rtneuron/SceneImpl.cpp	2017-03-01 15:52:59.236028000 +0000
+++ RTNeuron-78dca3d.new/rtneuron/rtneuron/SceneImpl.cpp	2017-03-01 15:34:00.512028000 +0000
@@ -491,7 +491,7 @@
         new MeshObject(vertices, primitive, colors, normals, attributes, this));
     {
         lunchbox::ScopedWrite mutex(_lock);
-        auto pair{std::make_pair(handler->getID(), handler)};
+        auto pair= std::make_pair(handler->getID(), handler);
         _meshObjects.insert(pair);
         _toAdd.insert(pair);
         ++_version;
@@ -508,8 +508,8 @@
 
         if (!_styleAttributeUpdates.empty())
         {
-            auto operation{std::make_shared<StyleAttributeUpdate>(
-                               _styleAttributeUpdates)};
+            auto operation = std::make_shared<StyleAttributeUpdate>(
+                               _styleAttributeUpdates);
             {
                 lunchbox::ScopedWrite opMutex(_operationsLock);
                 _masterSceneOperations->push(operation);
diff -ur '--exclude=CMake' RTNeuron-78dca3d/rtneuron/rtneuron/render/ViewStyle.cpp RTNeuron-78dca3d.new/rtneuron/rtneuron/render/ViewStyle.cpp
--- RTNeuron-78dca3d/rtneuron/rtneuron/render/ViewStyle.cpp	2017-03-01 15:52:59.488028000 +0000
+++ RTNeuron-78dca3d.new/rtneuron/rtneuron/render/ViewStyle.cpp	2017-03-01 15:36:25.488028000 +0000
@@ -464,7 +464,7 @@
 
 osgEq::SceneDecoratorPtr ViewStyle::clone() const
 {
-    auto viewStyle{std::make_shared<ViewStyle>()};
+    auto viewStyle = std::make_shared<ViewStyle>();
     viewStyle->getAttributes().merge(getAttributes());
     return viewStyle;
 }
diff -ur '--exclude=CMake' RTNeuron-78dca3d/rtneuron/rtneuron/scene/NeuronObject.cpp RTNeuron-78dca3d.new/rtneuron/rtneuron/scene/NeuronObject.cpp
--- RTNeuron-78dca3d/rtneuron/rtneuron/scene/NeuronObject.cpp	2017-03-01 15:52:59.580028000 +0000
+++ RTNeuron-78dca3d.new/rtneuron/rtneuron/scene/NeuronObject.cpp	2017-03-01 15:46:34.520028000 +0000
@@ -549,8 +549,8 @@
 
     void operator()(Scene::_Impl& scene) const
     {
-        auto handler{std::make_shared<ClientHandler>(
-                &scene, _objectID, _originID, _target)};
+        auto handler = std::make_shared<ClientHandler>(
+                &scene, _objectID, _originID, _target);
         if (_isDirty)
             handler->dirtyColoring();
         scene.insertObject(handler);
