diff -ur slurm-spank-plugins-0.23.old/lua/lua.c slurm-spank-plugins-0.23/lua/lua.c
--- slurm-spank-plugins-0.23.old/lua/lua.c	2012-03-30 23:08:04.000000000 +0200
+++ slurm-spank-plugins-0.23/lua/lua.c	2017-12-06 12:50:23.408768535 +0100
@@ -920,17 +920,17 @@
         return (0);
 
     if (level == -1) {
-        slurm_error (msg);
+        slurm_error ("%s", msg);
         lua_pushnumber (L, -1);
         return (1);
     }
 
     if (level == 0)
-        slurm_info (msg);
+        slurm_info ("%s", msg);
     else if (level == 1)
-        slurm_verbose (msg);
+        slurm_verbose ("%s", msg);
     else if (level == 2)
-        slurm_debug (msg);
+        slurm_debug ("%s", msg);
     return (0);
 }

