diff --git a/test/lib/CMakeLists.txt b/test/lib/CMakeLists.txt
index 3e9306c..b913d36 100644
--- a/test/lib/CMakeLists.txt
+++ b/test/lib/CMakeLists.txt
@@ -5,15 +5,25 @@ set(gtest_bin ${CMAKE_CURRENT_BINARY_DIR}/gtest)
 set(CMAKE_CXX_FLAGS_RELEASE "")
 set(CMAKE_CXX_FLAGS "-g ${CXX_DIALECT_OPT_CXX11} -O0")
 
-find_package(Subversion REQUIRED)
-file(MAKE_DIRECTORY "${gtest_src}")
-execute_process(COMMAND ${Subversion_SVN_EXECUTABLE} checkout "${gtest_repo}" "${gtest_src}")
-add_subdirectory(${gtest_src} ${gtest_bin})
+option(GTEST_USE_EXTERNAL "Use an external version of Gtest" FALSE)
+
+if(GTEST_USE_EXTERNAL)
+    find_package(GTest REQUIRED)
+    set(inc_dirs "${CMAKE_SOURCE_DIR}/src" "${GTEST_INCLUDE_DIRS}")
+    set(libs libsteps ${GTEST_BOTH_LIBRARIES})
+else()
+    find_package(Subversion REQUIRED)
+    file(MAKE_DIRECTORY "${gtest_src}")
+    execute_process(COMMAND ${Subversion_SVN_EXECUTABLE} checkout "${gtest_repo}" "${gtest_src}")
+    add_subdirectory(${gtest_src} ${gtest_bin})
+    
+    # tests are contained in sources test_XXX.cpp
+
+    set(inc_dirs "${CMAKE_SOURCE_DIR}/src" "${gtest_src}/include")
+    set(libs libsteps gtest_main)    
+endif()
 
-# tests are contained in sources test_XXX.cpp
 
-set(inc_dirs "${CMAKE_SOURCE_DIR}/src" "${gtest_src}/include")
-set(libs libsteps gtest_main)
 
 if (LAPACK_FOUND)
     list(APPEND libs ${LAPACK_LIBRARIES})
